docs/gallery/general/advanced_hdf5_io.py

```diff
@@ -216,6 +216,56 @@
 #   will be ignored as the h5py.Dataset will either be linked to or copied as on write.
 #
 
+####################
+# Parallel IO using MPI
+# -----------
+#
+# The HDF5 storage backend supports parallel I/O using the Message Passing Interface (MPI).
+# Using this feature requires that you install ``hdf5`` and ``h5py`` against an MPI driver, and you
+# install ``mpi4py``. The basic installation of pynwb will not work. Setup can be tricky, and
+# is outside the scope of this tutorial (for now), and the following assumes that you have
+# HDF5 installed in a MPI configuration. Here we:
+#
+# 1. Instantiate a dataset for parallel write
+#
+# 2. Write to that file in parallel using MPI
+#
+# 3. Read from the file in parallel using MPI.
+#
```

Comment:
I think it would help to expand the description of the operations a bit:

1. **Instantiate a dataset for parallel write:** We create TimeSeries with 4 timestamps that we will write in parallel
2. **Write to that file in parallel using MPI:** Here we assume 4 MPI ranks  while each rank writes the data for a different timestamp. 
3. **Read from the file in parallel using MPI:** Here each of the 4 MPI ranks reads one timestep from the file